subsys/net/ip/net_tls_default_credentials.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __NET_TLS_DEFAULT_CREDENTIALS_H__
+#define __NET_TLS_DEFAULT_CREDENTIALS_H__
+
+#define NET_TLS_DEFAULT_CA_CERTIFICATE_TAG 0
+#define NET_TLS_DEFAULT_PSK_TAG 1
+
+#if defined(MBEDTLS_X509_CRT_PARSE_C)
```

Comment:
Should we try to avoid having the certs in this format, meaning that parsing the cert requires that parsing functions are enabled in mbedtls. This takes valuable resources without much benefit. In net-app, the certs are required to be in DER (raw binary) format so that we can avoid this memory overhead. It should be quite trivial for the end user to convert the certs to DER format. What do you think?

Reply:
While I was looking for the certificates i saw they are typically provided in PEM format, so I 've used. But handling these is more resource consuming, that's a fact (AFAIR you have to enable it explicitly in mbedTLS). So yes, perhaps we should stick to the binary format in the samples, as it will save some bytes for a low cost, espacially I guess it will not require any changes in the code.

Reply:
Perhaps we could add a comment here, that if you want to use PEM you have to enable MBEDTLS_XXX expilcitly (and add this option to Kconfig)?

Reply:
> Perhaps we could add a comment here, that if you want to use PEM you have to enable MBEDTLS_XXX expilcitly (and add this option to Kconfig)?

Sure, that is a good idea.